index.js

```diff
@@ -1,11 +1,12 @@
+// @flow
 import { NativeModules } from 'react-native';
 
-export default (url) => {
+export const convertToProxyURL = (url: string) => string {
```

Comment:
I'm not sure if this is the correct flow syntax.

But if we do have flow syntax here, do we need to publish a version with them stripped out?

I'm not sure if everyone has flow setup.

Reply:
Same way I didn't want to write this one simple file in TypeScript. it would require extra step during publish.

btw, the d.ts change looks good. 👍